### PR TITLE
[Doc] Align ALTER SYSTEM SQL

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md [hunk 1]
```diff
@@ -2,178 +2,197 @@
...
-   ```plain text
-   broker: Name of broker
-   path: Remote storage path 
-   other properties: Other information required for accessing remote storage, e.g. Authentication information, etc. 
-   ```
+- Drop Broker nodes.
+
+> **CAUTION**
+>
+> Dropping a Broker nodes terminates the tasks currently running on it.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md:116]: a Broker node

File: docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md [hunk 2]
```diff
@@ -2,178 +2,197 @@
...
+
+```SQL
+ALTER SYSTEM CREATE IMAGE
+```
+
+Creating an image is an asynchronous operation on the Leader FE. You can check the start time and end time of the operation in the FE log file **fe.log**. A log like `triggering a new checkpoint manually...` indicates that the image creation has started, and a log like `finished save image...` indicates the image has been created.
+
+## Usage notes
+
+Adding and drooping FE, BE, CN, or Broker nodes are synchronous operations. You cannot cancel the node dropping operations.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md:148]: drooping --- dropping

File: docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md [hunk 3]
```diff
@@ -2,178 +2,197 @@
...
+  ALTER SYSTEM ADD FOLLOWER "<fe_host>:<edit_log_port>"[, ...]
+  ```
 
-   ```sql
-   ALTER SYSTEM ADD FREE BACKEND "host:heartbeat_port"[,"host:heartbeat_port"...]
-   ```
+  You can check the status of the new Follower FE by executing `SHOW PROC '/frontends'\G`.
 
-3. Add nodes to a cluster.
+- Drop a Follower FE.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md:23]: 1. drop follower 时要不要提示一下，follower 如果个数不够或者个数不对，对选主的影响。
2. 解释一下 drop be 和 decommission 的区别。drop 之后，节点处于什么状态，不能进行什么业务。
3. fe 是不存在 decommission 操作的吗？

File: docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md [hunk 4]
```diff
@@ -2,178 +2,201 @@
 
 ## Description
 
-This statement is used to operate nodes in a system. (Administrator only!)
+Manages FE, BE, CN, Broker nodes, and metadata snapshots in a cluster.
 
-## Syntax
+> **NOTE**
+>
+> Only the `root` user has the privilege to execute this SQL statement.
```
Comment (evelynzhaojie) [inline docs/sql-reference/sql-statements/Administration/ALTER SYSTEM.md:9]: 3.0 之后版本改为 Only the `cluster_admin` role has the privilege to execute this SQL statement.

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.3

**Pull request merged**